### ISSUE TITLE
Publish the generated app 

### ISSUE THREAD

**Issue by nicolas63** [jhipster/jhipster-dotnetcore#90]
one time per day ? or each commit ? publish the generated app to 
https://github.com/jhipster/jhipster-sample-app-dotnetcore

**Comment by A-5ingh** [comment]
I think we should publish every release.

**Comment by nicolas63** [comment]
Yes today i publish it manually each release i think we can automate this

**Comment by A-5ingh** [comment]
> Yes today i publish it manually each release i think we can automate this

Let me see if I can automate it if you are releasing today. I think we can use github workflow but then we will need a token that will allow to push code to generator app repository.

**Comment by A-5ingh** [comment]
@nicolas63 can you briefly exaplin what app do we publish to the repo https://github.com/jhipster/jhipster-sample-app-dotnetcore? and any special instructions to do so?

**Comment by nicolas63** [comment]
you can execute => jhipster-dotnetcore  generate-sample jwt-with-angular-app --skip-install --skip-checks --no-insight --with-entities for this repo https://github.com/jhipster/jhipster-sample-app-dotnetcore and jhipster-dotnetcore  generate-sample jwt-with-blazor-app --skip-install --skip-checks --no-insight --with-entities for this repo https://github.com/jhipster/jhipster-sample-app-blazor
If you need i can generate token and put it in github variable

**Comment by A-5ingh** [comment]
One challenge I see with above approach is that others repos ([jhipster-sample-app-dotnetcore](https://github.com/jhipster/jhipster-sample-app-dotnetcore) and [jhipster-sample-app-blazor](https://github.com/jhipster/jhipster-sample-app-blazor)) won't be able to know about when there is a new release in jhipster-dotnetcore. In this case we have to run the pipeline once a day to check if there are any new releases if yes then run generate commands otherwise skip.

Because I was thinking the other way around to trigger it on the new release in jhipster-dotnetcore repo and then push to other repos.

**Comment by nicolas63** [comment]
> Because I was thinking the other way around to trigger it on the new release in jhipster-dotnetcore repo and then push to other repos.

I think this is the best approach, and it's the one I was thinking.
